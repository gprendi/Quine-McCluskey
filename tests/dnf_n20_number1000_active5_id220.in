d 20
00010100010100000000
00010100010100000100
00010000010100000100
00000100000100000100
00010100010000000000
00000100010100000100
00000000010100000000
00010100000100000100
00010000000000000000
00010100010000000100
00010000000000000100
00010100000000000000
00010000010100000000
00010000010000000100
00000100010100000000
00000100010000000000
00000000010000000100
00000000010100000100
00000100000100000000
00010100000000000100
00000000000000000000
00010100000100000000
00000100010000000100
00000100000000000000
00010000010000000000
00000000000100000000
00000000010000000000
00000000000000000100
00000100000000000100
00010000000100000000
00010000000100000100
00000000000100000100
