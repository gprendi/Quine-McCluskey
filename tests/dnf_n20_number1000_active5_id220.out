p dnf 20 0
-1 -2 -3 -5 -7 -8 -9 -11 -13 -14 -15 -16 -17 -19 -20 0
